libs/common/src/common/auth.py

```diff
@@ -27,9 +27,9 @@ async def wrapper(*args, **kwargs):
     return wrapper
 
 
-def get_org_id_from_identity(identity):
+def assistant_user_id(identity):
     decoded_identity = base64.b64decode(identity).decode("utf8")
     identity_json = json.loads(decoded_identity)
-    identity = identity_json.get("identity", {})
-    org_id = identity.get("org_id")
-    return org_id
+    user_id = identity_json.get("identity").get("user").get("user_id")
+    org_id = identity_json.get("identity").get("org_id")
+    return f"{org_id}/{user_id}"
```

Comment:
This is similar to what we were doing with rasa right? Looking at this schema, the `user_id` can be null btw: [Here](https://github.com/RedHatInsights/identity-schemas/blob/main/3scale/schema.json#L64). So, in that situation, this will only amount to `org_id/None`.

Reply:
You're right, it looks like `user_id` is not defined for accounts of `type`: `System`. I'll use {orgid}/{system.cn} for those.

Reply:
Thanks @tahmidefaz ! Just updated. And [maybe found a bug](https://github.com/RedHatInsights/identity-schemas/pull/5) while reading about the identities.